### ISSUE TITLE
login is disabled

### ISSUE THREAD

**Issue by Aviortheking** [nextcloud/server#14471]
### Steps to reproduce
1. install nextcloud
2. try to launch it on chromium base browser
3. login button is disabled

### Expected behaviour
Tell us what should happen

the login button shouldn't be disabled

### Actual behaviour
Tell us what happens instead

the login button is disabled

### Server configuration

**Operating system**: idk

**Web server:** apache

**Database:** mysql

**PHP version:** 7.2.15

**Nextcloud version:** (see Nextcloud admin page) 15.0.4

**Updated from an older Nextcloud/ownCloud or fresh install:**
form older

**Where did you install Nextcloud from:** 
https://download.nextcloud.com/server/installer/setup-nextcloud.php

**Signing status:**
<details>
<summary>Signing status</summary>

```
Technical information
=====================
The following list covers which files have failed the integrity check. Please read
the previous linked documentation to learn more about the errors and how to fix
them.

Results
=======
- core
	- INVALID_HASH
		- .htaccess
- tasks
	- INVALID_HASH
		- l10n/fa.js
		- l10n/ru.js

Raw output
==========
Array
(
    [core] => Array
        (
            [INVALID_HASH] => Array
                (
                    [.htaccess] => Array
                        (
                            [expected] => 03a32baf7f3a61e407b4a452422c7641594cd7701426039443c2f10f311ca1a0d5e1865bbb8e7a0fba0c2144076f0af5781f95d78bdae4672ec2a6c08c338070
                            [current] => 9fef9dee376686f19407b1a952e00a98eed6a512ba128c0ceaeed24febcf44792cfcc7914b5c06ee1355b8f8df9f84dcf06a543daa3baa29fddf0d3d581e28f5
                        )

                )

        )

    [tasks] => Array
        (
            [INVALID_HASH] => Array
                (
                    [l10n/fa.js] => Array
                        (
                            [expected] => bd584b57921f379b128d9f4581fbeecfc2444e05390fa64b387614e927a8e2f5d6036a528efdc6e2b0e87f385bc53f5dcfbd32ce3d7ac332418e851a30bfeecf
                            [current] => 6e97ff850c34e4793ccceaca4d1baabbb1143f33470f3eaf7258db46b3c041ca6c2e682f1a36406c5516f5040a5d4965c59c7660b7271464f7a0bb47f5c9af75
                        )

                    [l10n/ru.js] => Array
                        (
                            [expected] => fed3cf349922c41fd979df347ef87b854a649e7db85f621ae4044bf7fffd40e55ded5d66beec823dfc5d60508b468889dba4cba3e8fbecfb748ac07ca26e83c4
                            [current] => 09b5fc177042f8c831666976e9cef7b3a8b9f91ab0e874a777c285262c5b7679a0dbba4120697c6a094048b4ac6894e6916f5edb5314a964bb931f08b4af32ea
                        )

                )

        )

)
```
</details>

**List of activated apps:**
<details>
<summary>App list</summary>

```

Activity
2.8.2
Official
AppOrder
0.6.0
Bookmarks
0.16.2
Brute-force settings
1.3.0
Official
Calendar
1.6.4
Official
Contacts
3.0.3
Official
Deleted files
1.5.0
Official
File sharing
1.7.0
Official
Gallery
18.2.0
Official
Log Reader
2.0.0
Official
Mail
0.11.0
Official
Markdown Editor
2.0.5
Monitoring
1.5.0
Official
Notes
2.5.1
Official
Notifications
2.3.0
Official
Passwords
2019.2.2
PDF viewer
1.4.0
Official
PhoneTrack
0.4.4
Tasks
0.9.8
Text editor
2.7.0
Official
Two Factor TOTP Provider
2.1.1
Official
Update notification
1.5.0
Official
Usage survey
1.3.0
Official
Versions
1.8.0
```
</details>

**Nextcloud configuration:**
<details>
<summary>Config report</summary>

```
<?php
$CONFIG = array (
  'passwordsalt' => '___',
  'secret' => '___',
  'trusted_domains' => 
  array (
    0 => '___',
  ),
  'datadirectory' => '/___/data',
  'dbtype' => 'mysql',
  'version' => '15.0.4.0',
  'overwrite.cli.url' => '___',
  'dbname' => '___',
  'dbhost' => '___',
  'dbport' => '',
  'dbtableprefix' => 'oc_',
  'dbuser' => '___',
  'dbpassword' => '___',
  'installed' => true,
  'theme' => 'nextcloud-breeze-dark',
  'loglevel' => 2,
  'maintenance' => false,
  'mail_smtpmode' => 'smtp',
  'mail_smtpsecure' => 'ssl',
  'mail_smtpauthtype' => 'LOGIN',
  'mail_from_address' => '___',
  'mail_domain' => '___',
  'mail_smtpauth' => 1,
  'mail_smtphost' => '___',
  'mail_smtpport' => '465',
  'mail_smtpname' => '___',
  'mail_smtppassword' => '___',
  'overwriteprotocol' => 'https',
  'updater.release.channel' => 'beta',
  'updater.secret' => '___',
  'instanceid' => '___',
);
```
</details>

**Are you using external storage, if yes which one:** no

**Are you using encryption:** no

**Are you using an external user-backend, if yes which one:** no

### Client configuration
**Browser:** Chrome, Brave, Chromium

**Operating system:** `Linux avior-neon 4.15.0-45-generic #48-Ubuntu SMP Tue Jan 29 16:28:13 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
` 

#### Web server error log
<details>
<summary>Web server error log</summary>

```
2a01:cb05:8596:4900:8cf2:6a3e:80a:c00 cloud.delta-wings.net - [17/Feb/2019:18:52:19 +0100] "GET /index.php/settings/ajax/checksetup HTTP/1.1" 200 1601 "-" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:65.0) Gecko/20100101 Firefox/65.0"
2a01:cb05:8596:4900:8cf2:6a3e:80a:c00 cloud.delta-wings.net - [17/Feb/2019:18:52:01 +0100] "GET /index.php/apps/logreader/poll?lastReqId=XGmc2goUFRYAAGee2c4AAAAi HTTP/1.1" 200 2 "-" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:65.0) Gecko/20100101 Firefox/65.0"
90.12.14.229 cloud.delta-wings.net - [17/Feb/2019:18:52:37 +0100] "GET /index.php/apps/bookmarks/public/rest/v2/bookmark?page=-1 HTTP/1.1" 200 25275 "-" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:65.0) Gecko/20100101 Firefox/65.0"
90.12.14.229 cloud.delta-wings.net - [17/Feb/2019:18:52:38 +0100] "GET /index.php/apps/bookmarks/public/rest/v2/folder HTTP/1.1" 200 2567 "-" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:65.0) Gecko/20100101 Firefox/65.0"
90.12.14.229 cloud.delta-wings.net - [17/Feb/2019:18:52:38 +0100] "GET /index.php/apps/bookmarks/public/rest/v2/folder/-1/childorder?layers=-1 HTTP/1.1" 200 3917 "-" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:65.0) Gecko/20100101 Firefox/65.0"
90.12.14.229 cloud.delta-wings.net - [17/Feb/2019:18:52:38 +0100] "PATCH /index.php/apps/bookmarks/public/rest/v2/folder/-1/childorder HTTP/1.1" 200 20 "-" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:65.0) Gecko/20100101 Firefox/65.0"
90.12.14.229 cloud.delta-wings.net - [17/Feb/2019:18:52:39 +0100] "PATCH /index.php/apps/bookmarks/public/rest/v2/folder/7/childorder HTTP/1.1" 200 20 "-" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:65.0) Gecko/20100101 Firefox/65.0"
90.12.14.229 cloud.delta-wings.net - [17/Feb/2019:18:52:39 +0100] "PATCH /index.php/apps/bookmarks/public/rest/v2/folder/13/childorder HTTP/1.1" 200 20 "-" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:65.0) Gecko/20100101 Firefox/65.0"
90.12.14.229 cloud.delta-wings.net - [17/Feb/2019:18:52:39 +0100] "PATCH /index.php/apps/bookmarks/public/rest/v2/folder/15/childorder HTTP/1.1" 200 20 "-" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:65.0) Gecko/20100101 Firefox/65.0"
90.12.14.229 cloud.delta-wings.net - [17/Feb/2019:18:52:40 +0100] "PATCH /index.php/apps/bookmarks/public/rest/v2/folder/19/childorder HTTP/1.1" 200 20 "-" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:65.0) Gecko/20100101 Firefox/65.0"
2a01:cb05:8596:4900:8cf2:6a3e:80a:c00 cloud.delta-wings.net - [17/Feb/2019:18:52:20 +0100] "GET /index.php/apps/logreader/poll?lastReqId=XGmc2goUFRYAAGee2c4AAAAi HTTP/1.1" 200 2 "-" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:65.0) Gecko/20100101 Firefox/65.0"
90.12.14.229 cloud.delta-wings.net - [17/Feb/2019:18:52:40 +0100] "PATCH /index.php/apps/bookmarks/public/rest/v2/folder/21/childorder HTTP/1.1" 200 20 "-" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:65.0) Gecko/20100101 Firefox/65.0"
2a01:cb05:8596:4900:8cf2:6a3e:80a:c00 cloud.delta-wings.net - [17/Feb/2019:18:52:49 +0100] "GET /ocs/v2.php/apps/notifications/api/v2/notifications HTTP/1.1" 200 74 "-" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:65.0) Gecko/20100101 Firefox/65.0"
2a01:cb05:8596:4900:8cf2:6a3e:80a:c00 cloud.delta-wings.net - [17/Feb/2019:18:52:49 +0100] "GET /apps/notifications/img/notifications.svg HTTP/1.1" 304 - "-" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:65.0) Gecko/20100101 Firefox/65.0"
2a01:cb05:8596:4900:8cf2:6a3e:80a:c00 cloud.delta-wings.net - [17/Feb/2019:18:52:40 +0100] "GET /index.php/apps/logreader/poll?lastReqId=XGmc2goUFRYAAGee2c4AAAAi HTTP/1.1" 200 2 "-" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:65.0) Gecko/20100101 Firefox/65.0"
2a01:cb05:8596:4900:8cf2:6a3e:80a:c00 cloud.delta-wings.net - [17/Feb/2019:18:53:19 +0100] "GET /ocs/v2.php/apps/notifications/api/v2/notifications HTTP/1.1" 200 74 "-" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:65.0) Gecko/20100101 Firefox/65.0"
2a01:cb05:8596:4900:8cf2:6a3e:80a:c00 cloud.delta-wings.net - [17/Feb/2019:18:53:20 +0100] "GET /apps/notifications/img/notifications.svg HTTP/1.1" 304 - "-" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:65.0) Gecko/20100101 Firefox/65.0"
2a01:cb05:8596:4900:8cf2:6a3e:80a:c00 cloud.delta-wings.net - [17/Feb/2019:18:53:00 +0100] "GET /index.php/apps/logreader/poll?lastReqId=XGmc2goUFRYAAGee2c4AAAAi HTTP/1.1" 200 2 "-" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:65.0) Gecko/20100101 Firefox/65.0"
2a01:cb05:8596:4900:8cf2:6a3e:80a:c00 cloud.delta-wings.net - [17/Feb/2019:18:53:21 +0100] "GET /index.php/apps/logreader/poll?lastReqId=XGmc2goUFRYAAGee2c4AAAAi HTTP/1.1" 200 2 "-" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:65.0) Gecko/20100101 Firefox/65.0"
2a01:cb05:8596:4900:8cf2:6a3e:80a:c00 cloud.delta-wings.net - [17/Feb/2019:18:53:49 +0100] "GET /ocs/v2.php/apps/notifications/api/v2/notifications HTTP/1.1" 200 74 "-" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:65.0) Gecko/20100101 Firefox/65.0"
2a01:cb05:8596:4900:8cf2:6a3e:80a:c00 cloud.delta-wings.net - [17/Feb/2019:18:53:49 +0100] "GET /apps/notifications/img/notifications.svg HTTP/1.1" 304 - "-" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:65.0) Gecko/20100101 Firefox/65.0"
2a01:cb05:8596:4900:8cf2:6a3e:80a:c00 cloud.delta-wings.net - [17/Feb/2019:18:53:41 +0100] "GET /index.php/apps/logreader/poll?lastReqId=XGmc2goUFRYAAGee2c4AAAAi HTTP/1.1" 200 2 "-" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:65.0) Gecko/20100101 Firefox/65.0"
2a01:cb05:8596:4900:fc7b:7ccf:207b:3ff2 cloud.delta-wings.net - [17/Feb/2019:18:54:17 +0100] "GET /index.php/apps/phonetrack/api/getsessions HTTP/1.1" 200 154 "-" "phonetrack-android/0.0.8"
2a01:cb05:8596:4900:8cf2:6a3e:80a:c00 cloud.delta-wings.net - [17/Feb/2019:18:54:19 +0100] "GET /ocs/v2.php/apps/notifications/api/v2/notifications HTTP/1.1" 200 74 "-" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:65.0) Gecko/20100101 Firefox/65.0"
2a01:cb05:8596:4900:8cf2:6a3e:80a:c00 cloud.delta-wings.net - [17/Feb/2019:18:54:19 +0100] "GET /apps/notifications/img/notifications.svg HTTP/1.1" 304 - "-" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:65.0) Gecko/20100101 Firefox/65.0"
2a01:cb05:8596:4900:fc7b:7ccf:207b:3ff2 cloud.delta-wings.net - [17/Feb/2019:18:54:22 +0100] "GET /index.php/apps/phonetrack/api/getsessions HTTP/1.1" 200 154 "-" "phonetrack-android/0.0.8"
2a01:cb05:8596:4900:8cf2:6a3e:80a:c00 cloud.delta-wings.net - [17/Feb/2019:18:54:02 +0100] "GET /index.php/apps/logreader/poll?lastReqId=XGmc2goUFRYAAGee2c4AAAAi HTTP/1.1" 200 2 "-" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:65.0) Gecko/20100101 Firefox/65.0"
2a01:cb05:8596:4900:8cf2:6a3e:80a:c00 cloud.delta-wings.net - [17/Feb/2019:18:54:23 +0100] "GET /index.php/apps/logreader/poll?lastReqId=XGmc2goUFRYAAGee2c4AAAAi HTTP/1.1" 200 2 "-" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:65.0) Gecko/20100101 Firefox/65.0"
2a01:cb05:8596:4900:8cf2:6a3e:80a:c00 cloud.delta-wings.net - [17/Feb/2019:18:54:49 +0100] "GET /ocs/v2.php/apps/notifications/api/v2/notifications HTTP/1.1" 200 74 "-" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:65.0) Gecko/20100101 Firefox/65.0"
2a01:cb05:8596:4900:8cf2:6a3e:80a:c00 cloud.delta-wings.net - [17/Feb/2019:18:54:49 +0100] "GET /apps/notifications/img/notifications.svg HTTP/1.1" 304 - "-" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:65.0) Gecko/20100101 Firefox/65.0"
2a01:cb05:8596:4900:8cf2:6a3e:80a:c00 cloud.delta-wings.net - [17/Feb/2019:18:54:43 +0100] "GET /index.php/apps/logreader/poll?lastReqId=XGmc2goUFRYAAGee2c4AAAAi HTTP/1.1" 200 2 "-" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:65.0) Gecko/20100101 Firefox/65.0"
95.24.0.153 delta-wings.net - [17/Feb/2019:18:52:42 +0100] "GET / HTTP/1.1" 200 3008 "https://yandex.ru/clck/jsredir?from=yandex.ru%3Bsearch%3Bweb%3B%3B&text=&etext=2067.5UQ3wh_NESRk5wwEUAQod27PBaU0KF-4jDly36nZoMnnkdy4Q8xZKhM39QdAKdAO.7e9c06c93027eb068b0aea046f6b3dd5babc4ee5&uuid=&state=_BLhILn4SxNIvvL0W45KSic66uCIg23qh8iRG98qeIXmeppkgUc0YGHEttDsYq_w3pSJtejupWg&data=UlNrNmk5WktYejR0eWJFYk1LdmtxclhPemhuT0pDWTY0Z0k3TldPLWRlazMxOTJRNDBTdlVna09ZUUZQckYtT2Zzd2FTWUw5Nms2V29zVElzalRFSzQ2YzcyU2RYMmNObWRGbE1FcDR1NnM&b64e=2&sign=b889371c5f366e87fe2836bf024f8c9b&keyno=0&cst=AiuY0DBWFJ7IXge4WdYJQa9gkHE_3kbimmkHEgl5OrFLqHf--6tkSrZhbXN7qySRJzXQMdNAAZKCm7s6k1ZMY9GJCJz3r8UHit4gdLZlotcqz6Wr8EFIKqK7TJmZiJqTyph_3Rf9LBa1uu02TjMDSSbAsOmwVp37&ref=orjY4mGPRjk5boDnW0uvlrrd71vZw9kp5uQozpMtKCUDAXrtBVCJpNjcIbw2dqU5hwvl0fVIJT_j01xmeUxEfblJfCE9qH1xpgVwKaJhIhfoKtZs3qomsA&l10n=ru&cts=1550414589202&mc=5.37118867982" "Mozilla/5.0 (compatible; MSIE 10.0; Windows NT 6.2; WOW64; Trident/6.0; Touch; MASMJS)"
95.24.0.153 delta-wings.net - [17/Feb/2019:18:52:42 +0100] "GET /assets/000ca93373ce5ea622b5016c0bf32cc7.css?g-d357bd84 HTTP/1.1" 200 5526 "http://delta-wings.net/" "Mozilla/5.0 (compatible; MSIE 10.0; Windows NT 6.2; WOW64; Trident/6.0; Touch; MASMJS)"
```
</details>

#### Nextcloud log (data/nextcloud.log)
<details>
<summary>Nextcloud log</summary>
[nextcloud.log](https://github.com/nextcloud/server/files/2922115/nextcloud.log)
</details>

#### Browser log
<details>
<summary>Browser log</summary>

```
a) The javascript console log
GET https://cloud.delta-wings.net/index.php/login?redirect_url=/index.php/apps/files/ net::ERR_SPDY_PROTOCOL_ERROR 200
b) The network log
idk how to get thoses logs
```
</details>

**Comment by ChristophWurst** [comment]
Could you please post a screenshot?

**Comment by Aviorleking** [comment]
![screenshot_20190304-170204_brave](https://user-images.githubusercontent.com/15822031/53745462-53503b80-3e9f-11e9-8910-b9eb82b42ecf.png)
Exemple on brave for Android

**Comment by ChristophWurst** [comment]
Is that login button really disabled? Looks more like you're using a grey theme.

**Comment by Aviorleking** [comment]
nop it's disabled in the html element there is a disabled="disabled" element and when it's not it's a blue one and when I click I change tha page

**Comment by Aviorleking** [comment]
without the theme loaded
![image](https://user-images.githubusercontent.com/15822031/53764779-83630300-3ece-11e9-8c90-83ccb1156282.png)

**Comment by ChristophWurst** [comment]
Could you check your console? Are there any errors? Did you block javascript?

**Comment by Aviorleking** [comment]
I don't block any js on my webpages

the console return this
```log
GET https://cloud.delta-wings.net/index.php/login net::ERR_SPDY_PROTOCOL_ERROR 200
```

**Comment by Aviorleking** [comment]
now I can say that this error only append when nextcloud is installed on a host and for me in mutualized
I reinstalled not long ago my nextcloud instance on a VPS an it work perfectly !
Today a installed a new instance on another OVH mutualized and after passing the login button i GOT the same as before but this time in the first page for like 10 - 15 minutes and after that my new instance seems to work. but still for some reason it don't work like it should.

**Comment by Napafaitlix** [comment]
I think I have the same issue. I am also using a shared hosting of OVH. I had the problem on NC14. I tried to update to 15 and then 16, but the problem is still here. I also tried PHP 7.1 and 7.3. Same issue.
Chrome 74.0.3729.169 (64 bits) throws an error:
`Error parsing header X-XSS-Protection: 1; mode=block, 1; mode=block: expected semicolon at character position 13. The default protections will be applied.`

and also:

`Failed to load resource: net::ERR_SPDY_PROTOCOL_ERROR `

It seems to work better with Firefox 67.0.2 (64 bits) since I can log in, at least, but the console shows an other error:

```
JQMIGRATE: Migrate is installed, version 1.4.1 jquery-migrate.min.js:2:494
Échec du chargement pour l’élément <script> dont la source est « https://owncloud.xyz.fr/index.php/apps/theming/js/theming?v=0 ». login:55:1
SyntaxError: missing } after function body
accessibility:5:2note: { opened at line 1, column 12accessibility:1:12
```

Any idea? Did one of you solved the problem or knows what OVH should set on their side to fix it?

**Comment by Aviortheking** [comment]
Ovh is blocking something :thinking:

**Comment by skjnldsv** [comment]
Any news from ovh?
Can we have access to the login page maybe? So we can check ourselves?

**Comment by vladbejenaru** [comment]
I have the same problem. I run nextcloud in a docker from linuxserver.io and my problem is only on the computer from where I have make the installation (with all the trial and error on the process). All the other clients, browsers or android apps are functioning without any problem. I suspect a cache problem, but what is funny is that the problem appears on all the browsers on this computer even after cleaning local cache... The computer is running Xubuntu 1904... anybody... any idea?

**Comment by floviolleau** [comment]
Hi,

I think I have the same issue that @Napafaitlix raised

I am also using a shared hosting of OVH. I'm in Nextcloud 16.0.4 and Php 7.3 Stable production

76.0.3809.132 (Official Build) (64-bit) throws an error:
Error parsing header X-XSS-Protection: 1; mode=block, 1; mode=block: expected semicolon at character position 13. The default protections will be applied.

It is working fine with Firefox 69.0 (64 bits) since I can log in, but the JS console shows an different error:

```
JQMIGRATE: Migrate is installed, version 1.4.1 main.js:278:531
Échec du chargement pour l’élément <script> dont la source est « https://my-domain.fr/index.php/apps/theming/js/theming?v=3 ». login:53:1
SyntaxError: missing } after function body accessibility:5:2
note: { opened at line 1, column 12 accessibility:1:12
```

Any clues?

**Comment by taynted** [comment]
I experienced the exact same problems on a SpinupWP server. I fixed it by commenting out css and js caching in 
`/etc/nginx/sites-available/sub.domain.tld/server/static-files.conf`

The nginx doc led me in the right direction.
[https://docs.nextcloud.com/server/stable/admin_manual/installation/nginx.html?highlight=nginx#javascript-js-or-css-css-files-not-served-properly](url)

**Comment by floviolleau** [comment]
Hi,

Thanks for the help here but I'm not able to change the configuration of the web server...

This is not allowed by the provide.

KR

**Comment by kesselb** [comment]
Can someone with a broken instance share the url? You can also mail it to mail@kesselb.dev

**Comment by fabienli** [comment]
Hi @floviolleau, 

I'm running nextcloud 16.0.4 on a mutualized ovh. 
I had similar issue as described above (login issue on Chrome) which was solved when adding this in my .htaccess:
```
<IfModule mod_deflate.c>
SetEnv no-gzip 1
</IfModule>
```

note that on firefox, I also have the following error reported in that thread:

> SyntaxError: missing } after function body            accessibility:5:2
> note: { opened at line 1, column 12

but this seems not to have any impact on login and using nextcloud (I may missed some accessibility feature, but I've not been able to check this further yet)

**Comment by floviolleau** [comment]
Wow! @fabienli Huge thanks! You saved me from the big headache I had since my users was bugging me on that issue!

This exactly solved the issue.

I don't know with which hints you found the solution but thanks a lot! It is working fine.

I can say that it solved as well #17246

I still have the issue with firefox but like you, it's seems not to have any impact on login and using nextcloud.

Thanks!

**Comment by fabienli** [comment]
To complement on the issue, it seems that OVH sends invalid header when using compression.
(more details here: https://forums.bitfire.at/topic/1218/davdroid-not-working-with-nextcloud/9 )

It seems that the solution is not only helping for davdroid but also in this case!

**Comment by floviolleau** [comment]
Thanks for the link.

**Comment by joselito11** [comment]
Can I ask here? How to reset password? Can not login anymore

**Comment by floviolleau** [comment]
Hi,

Maybe you can reset here the password by using this documentation: https://docs.nextcloud.com/server/15/admin_manual/configuration_user/reset_admin_password.html

KR

**Comment by joselito11** [comment]
As I write before, with these command is no go: `Could not open input file: /var/www/nextcloud/occ`

**Comment by kesselb** [comment]
> Can I ask here? 

@joselito11 Please don't hijack this thread. https://help.nextcloud.com is the place to ask questions.

**Comment by kesselb** [comment]
Did some testing :eyes: Thanks @taynted for the test instance :+1: I share my findings here it might be helpful for someone else.

> Error parsing header X-XSS-Protection: 1; mode=block, 1; mode=block: expected semicolon at character position 13. The default protections will be applied.

Some headers are sent by nextcloud and the webserver. This leads to the situation that the values are appended. For example the x-xss-protection is there two times so the value will be 1; mode=block, 1; mode=block which is invalid. There is another issue for this kind of problem: https://github.com/nextcloud/server/issues/8207

Please ensure that there are not duplicate headers. You can tell nextcloud to not send the headers by setting `modHeadersAvailable` to `true` within your nginx or apache2 configuration.

> Login button is disabled.

https://github.com/nextcloud/server/blob/1c1cd10f0d8c7e6a11bb3d7ae2b476b8e3df74f8/core/templates/login.php#L58

The input is disabled by default (for nextcloud 16). It will be enabled by javascript somewhere. 

![image](https://user-images.githubusercontent.com/3902676/65715528-907b5580-e09d-11e9-9c3c-273db6a111bd.png)

The network request for the login page looks like above. Two resources are not found. They are not found because they don't exist as physical files in the filesystem and generated by nextcloud on request. A special webserver configuration is required for this. For apache2 it's part of the .htaccess file and for Nginx documented here: https://docs.nextcloud.com/server/17/admin_manual/installation/nginx.html

```
    location ~ \.(?:css|js|woff2?|svg|gif|map)$ {
        try_files $uri /index.php$request_uri;
    }
```

Above block check if there is a file with that name (direct hit) and if not the request is forwarded to index.php and nextcloud will handle the request.

**tl;dr If you run into a similar issue please check your configuration. Unfortunately there are many error sources. Please ensure that there are no errors within your browser console or network tab.**
